readme.md

```diff
@@ -94,7 +103,7 @@ How is the concept of OAuth related to a valet key?
 
 Your Answer:
 ```text
-
+Like a valet key, OAuth uses a standard protocol to authenticate and grant access but doesn't expose everything on the server and database to the third party used for authentication. 
```

Comment:
:+1: Well put! Also the party authenticating (your application) is not given full access to the user's third party account